src/gui/toolbarMenubar/ToolPageLayer.h

```diff
@@ -18,8 +18,6 @@
 
 #include "AbstractToolItem.h"
 
-using std::map;
-
```

Comment:
As this has not really anything to do with the rest of your PR, please revert this and create a separate PR for this.